__tests__/components/sections/bounties/Navigation.test.tsx

```diff
@@ -0,0 +1,65 @@
+import "@testing-library/jest-dom";
+import { screen } from "@testing-library/react";
+import BountiesNavigation from "@/components/sections/bounties/Navigation";
+import { renderWithRedux } from "../../../../__mocks__/renderWithRedux";
+import { List } from "@/utilities/CommunityNavigation";
+
+jest.mock("next/router", () => ({
+  useRouter: () => ({
+    query: "/",
+  }),
+}));
+
+const expectedNavItems: Omit<List, "id">[] = [
+  {
+    title: "bounties.navigation",
+    items: [
+      {
+        label: "bounties.navigation.all",
+        exact: true,
+        link: "/bounties",
+      },
+    ],
+  },
+];
+
+describe("Navigation", () => {
+  it("should render the navigation", () => {
+    renderWithRedux(<BountiesNavigation />);
+    const bountiesNavigation = screen.getByTestId("bountiesNavigationId");
+    expect(bountiesNavigation).toBeInTheDocument();
+  });
+
+  it("should render menu items", () => {
+    renderWithRedux(<BountiesNavigation />);
+    const menuItems = screen.getAllByRole("listitem");
+    menuItems.forEach(() => {
+      expectedNavItems.forEach((menu) => {
+        if (!menu.hideTitle) {
+          const menuTitle = menu.title;
+          expect(screen.getByText(menuTitle)).toBeInTheDocument();
```

Comment:
Shouldn't only check the element to be in the document you can also check for in the title is the one you mocked